src/redist/targets/GenerateLayout.targets

```diff
@@ -166,14 +166,14 @@
 
       <BundledInstallerComponent Include="DownloadedNetStandardTargetingPackInstallerFile"
                            Condition="'$(SkipBuildingInstallers)' != 'true' And '$(InstallerExtension)' != '' And !$(Architecture.StartsWith('arm'))">
-        <BaseUrl>$(CoreSetupRootUrl)$(MicrosoftNETCoreAppRuntimewinx64PackageVersion)</BaseUrl>
+        <BaseUrl>$(CoreSetupRootUrl)$(MicrosoftNETCoreAppPackageVersion)</BaseUrl>
```

Comment:
Looks like HostFxrVersion and SharedHostVersion are stabilized still. They come from MicrosoftNETCoreDotNetHostResolverPackageVersion and MicrosoftNETCoreDotNetAppHostPackageVersion